ami/ml/tasks.py

```
@@ -15,12 +15,21 @@ def process_source_images_async(pipeline_choice: str, endpoint_url: str, image_i
-        logger.error(f"Job {job_id} not found: {e}")
-        pass
+    reprocess_all_images = False
+    if job_id is not None:
+        try:
+            job = Job.objects.get(pk=job_id)
+            reprocess_all_images = job.project.feature_flags.reprocess_all_images
+            job.logger.info(
+                f"Processing {len(image_ids)} images for job {job} (reprocess_all_images={reprocess_all_images})"
+            )
+        except Job.DoesNotExist as e:
+            logger.error(f"Job {job_id} not found: {e}")
```

Ruff (0.14.5)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)